airflow/config_templates/default_airflow.cfg

```diff
@@ -716,6 +716,9 @@ audit_view_excluded_events = gantt,landing_times,tries,duration,calendar,graph,g
 # Example: audit_view_included_events = dagrun_cleared,failed
 # audit_view_included_events =
 
+# Logical condition for tags filtering (AND or OR).
+dags_tags_filter_condition = AND
```

Comment:
i don't think having this as setting is the best solution.
This feels more like something that we should solve in UX level

Reply:
I agree; see my comments here: https://github.com/apache/airflow/issues/10724#issuecomment-687609559
The idea is to have a config as the default behavior (which should be OR to be backward compatible) and add a UI select box, but I don't have time to work on the UI (and it has changed a lot since I worked on it...), so I want to get this feature in and then maybe someone else could add the UI part.

Reply:
I agree. If this is something we want to support then it should be user-selected at the start with 0 config.